packages/react-components/lib/map/three-fiber/circle-shape.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { CircleShape } from './circle-shape';
+import { Euler, Vector3 } from 'three';
+import { makeRobotData } from '../test-utils.spec';
+import { Canvas } from '@react-three/fiber';
+
+describe('CircleShape', () => {
+  it('debería renderizar un círculo y una línea correctamente', () => {
```

Comment:
We should probably keep descriptions in english

Reply:
Thanks for catching it!
Done cc14bed4b0ee59f55fc6efe1f29c520a1eee9028!